### DISCUSSION TITLE
How to set up a transceiver?

### DISCUSSION THREAD

**Discussion by caffeinism** [aiortc/aiortc#1027] [Q&A]
Hi, I'm just a regular user who is finding this project useful.

I'm looking for a way to set the appropriate codec and bitrate for the media I want to send. What I've found is that in the code, it's encoded through a CodecContext that is set to the default in the Codec, which is internally wrapped in PyAV.

However, I don't think this is sufficiently usable. Is there any way to select the appropriate codec and encoding parameters, bitrate, etc.? It would be nice to be able to provide the application with the appropriate options.

**Comment by bherbruck** [comment]
@caffeinism did you ever figure it out?
